apps/src/templates/progress/progressHelpers.js

```diff
@@ -104,6 +104,20 @@ export function isLevelAssessment(level) {
   return level.kind === 'assessment';
 }
 
+/**
+ * Checks if a whole stage is assessment levels
+ * @param {[]} levels An array of levels
+ * @returns {bool} If all the levels in a stage are assessment levels
+ */
+export function stageIsAllAssessment(levels) {
+  for (let i = 0; i < levels.length; i++) {
+    if (levels[i].kind !== LevelKind.assessment) {
+      return false;
+    }
+  }
+  return true;
```

Comment:
this method can be implemented more concisely using standard libraries:
```js
function stageIsAllAssessment(levels) {
  return levels.every(level => level.kind === LevelKind.assessment);
}
```

this is the best reference I've found for what's available: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array